.gitignore

```diff
@@ -6,3 +6,4 @@ dist/*.js
 dist/*.css
 dist/img/*.svg
 data/data.js
+js/lib/id/*.js
```

Comment:
Okay. I totally got it wrong.
As per our conversation last week weren't we planning to ignore the bundles
generated by rollup ?

On Wed, Jul 6, 2016 at 4:48 PM Bryan Housel <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/openstreetmap/iD/pull/3229#discussion_r69712807>:
>
> > @@ -6,3 +6,4 @@ dist/*.js
> >  dist/*.css
> >  dist/img/*.svg
> >  data/data.js
> > +js/lib/id/*.js
>
> To preserve the ability for people to check out the project and use it
> without a build step, I think we want to keep js/lib/id/index.js checked
> in for now. Even though it is a generated file, it's used by index.html
> in project root.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/openstreetmap/iD/pull/3229/files/39c82af15891bc29f89502ed60a3498895a1353e..3ec8c5013782edd536c4fd2b6951097256591852#r69712807>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGpL7iCw4WjgvCT1Q9vGQCPsaEUqrk-3ks5qS471gaJpZM4JFy6u>
> .
>